### ISSUE TITLE
Reconsider `rights` in light of `requiredStatement`

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1479]
Given 
  * the change in scope from just a license to also include a rights statement, 
  * the inability to associate an internationalized label, per the change to `attribution`,
  * that requiredStatement can use HTML to include links
  * that the URI is not necessarily machine readable / actionable,
  * it's unclear as to the scope of the assertion (just this resource, all included resources, even from other organizations),
  * and it's the _Presentation_ API ...

I think we should consider dropping `rights` in favor of just putting such content into `requiredStatement` as HTML.

**Comment by zimeon** [comment]
Are there institutions for whom this change would make the 2->3 transition a legal problem (requiring more discussion about acceptable right statements etc.) instead of just a technical problem?

**Comment by azaroth42** [comment]
It would have the same amount of lossiness as related -> homepage. If there was both rights and attribution, they would both go into requiredStatement as HTML by a conversion routine. 

An alternative would be to tighten the semantics up a bit to make it clear that it is the license / rights under which the current document may be re-used.  e.g. You can or can't create derivatives of this manifest or image.  At the moment it's fuzzy as to whether it's the work or the Presentation API document.

**Comment by zimeon** [comment]
I think `rights` as a simple list of URIs for things like CC, rightsstatements.org (as defined in Image API now) has utility separate from `requiredStatement` so I favor keeping it. Questions then are:

  * can we simplify like that in prezi?
  * can we define what the rights statement applies to well enough for it to make good sense in prezi?

**Comment by irv** [comment]
I agree with @zimeon that the URI is useful on its own.

I don't get the interpretation that `license`/`rights` applies to the manifest rather than the representaton of the content, though. My well thought out and carefully reasoned argument is this: you can't assert auth on the manifest json, but you can on the resources.

**Comment by aisaac** [comment]
There is quite some uncertainty about what `rights` attached to a manifest would be. It can be either about the manifest as a representation/data, or about the object that the manifests represents (a book). It's a bit like the http-range-14 discussion (in the sense that IIIF says a manifest is a representation, but the label property in it should be like "this book" and not "manifest about this book". 

But in any case, there is a rights level that's not captured by rights in the Image API alone. It is desirable that a machine-readable representation of rights can be found, and it seems dangerous to remove it without having a suitable best practice to alleviate its removal. And I expect that coming with this would take much time. At this stage I would prefer that IIIF leaves options open. If just because Europeana plans to use it :-)

**Comment by azaroth42** [comment]
Eds' call:

Proposal 1:  Reduce `rights` to only be on content resources, mirroring the use in the Image API.  

Proposal 2: Make it much clearer that it refers only to the representation, e.g. the JSON not the object.

Open issue as to the value -- object vs uri, and cardinality.

**Comment by aisaac** [comment]
+1 for proposal 1

**Comment by workergnome** [comment]
It looks like proposal 1 says that this would only be applicable to the target of annotations...what about canvases that are treated as content resources?

**Comment by azaroth42** [comment]
Proposal 1 would be any external resource, including the values of `seeAlso`, `rendering`, and so forth.  What's the use case for a license on a Canvas, rather than the content resources that paint it?

**Comment by zimeon** [comment]
[`profile` in Prezi 3](http://prezi3.iiif.io/api/presentation/3.0/#profile) admits a single URI value like [`license` in Image 2.1](http://iiif.io/api/image/2.1/#rights-and-licensing-properties) -- would it really be too evil to do that for `rights` in both Image and Prezi? Having to describe rights statement URIs as a resource with a type would be problematic for URIs that are designed to content negotiate, such as rightsstatements.org:

```
simeon@Cider ~> curl -I http://rightsstatements.org/vocab/InC/1.0/
HTTP/1.1 303 See Other
Location: http://rightsstatements.org/page/InC/1.0/?language=en

simeon@Cider ~> curl -I -H "Accept: application/json+ld" http://rightsstatements.org/vocab/InC/1.0/
HTTP/1.1 303 See Other
Location: http://rightsstatements.org/data/InC/1.0/
```

**Comment by azaroth42** [comment]
If the rights statement is a flag like profile is, and doesn't need to be resolved, then sure. But I don't think that's the case always.  In the best situation there's a CC or rightsstatements URI ... all good. But when there isn't, then what?

**Comment by azaroth42** [comment]
#1284 proposes that URIs be used by themselves only when the space is enumerable.  If we limit the rights statement URIs to ONLY CC and rightsstatements.org then they can be plain URIs. Otherwise they should be JSON objects.

**Comment by zimeon** [comment]
perhaps: registered URIs? they are for community interoperability after all -- though I do worry that we may be creating a registry headache

**Comment by azaroth42** [comment]
Well, again, what's the intent of including the information?  

I think there are two realistic possibilities:
(a) That the client will see the URI and know what to do. Maybe that is just present the right icon somewhere, or maybe it tries to prevent the user from doing some action, but either way it's a flag that affects the processing of the resource.
(b) That the client will display the URI to the user and let them click on it to see what they are allowed to do.

If it's (b), then we have `requiredStatement` for that to solve the label and internationalization issues. If it's (a) then we need to limit and describe the required interactions.

If it's (c), it's only there to make publishers feel better about things, but no one actually does anything with it, even display it, then I think that's grounds to drop it.

**Comment by zimeon** [comment]
I think
(d) rights URIs enable systems to make choices about reuse, or to present options to users about reuse (like the reuse pull-downs on Flickr or Google Image search)

**Comment by azaroth42** [comment]
I don't think that they do. 

"http://example.org/foo/bar/baz" ... what do you do with that? Just put that exact string in front of the user? We know where that leads ... `requiredStatement`. To me, your (d) is my (a).

**Comment by zimeon** [comment]
I do use the rights/reuse pull-downs on Flickr and Google Image search -- these seem like worthwhile end-user features to enable (which would not be enabled with `requiredStatement` unless I'm missing something).

These only work if folks use a limited number of right statement tokens (URIs) from well-established vocabs. An arbitrary URI such as "http://example.org/foo/bar/baz" would be ignored, but surely we don't want to hard-code the vocabulary in a versioned spec? Perhaps missing the needs of a key community?

**Comment by azaroth42** [comment]
Okay, I think we agree on a/d and that the implication is that the set of possible URIs must be enumerable. Which means either limiting to external sets (rs, cc) or allowing the community to register their own additional ones.  

The requirement is that the user must be made aware of the set of rights/licenses.

**Comment by zimeon** [comment]
I agree with first part, not sure that the user needs to be made aware: On an individual resource basis that is the role of `requiredStatement`, I see `rights` as facilitating features selecting or UI features for set of resources so it seems optional whether a UI showing that one resource does anything with it at all

**Comment by kestlund** [comment]
I want to +1 the a and d use cases.

Are there issues with defining CC and Rightstatements.org in a cookbook or registry and then updating the registry rather than the spec as use-cases emerge?

**Comment by azaroth42** [comment]
Community Call 2018-03-14:  

Consensus on way forwards:

`rights` is an enumerated list of values with the initial set being the union of CC and rightsstatements.org.  Create a registry for community additions. Clients can ignore values that they do not understand.  If the publisher requires the information to be shown, or to be shown in a certain way, then that must go into `requiredStatement`.

**Comment by azaroth42** [comment]
After further discussion on slack, and on the editors' call, eds propose:

A string that identifies a license or rights statement that applies to the content of this resource, such as the JSON of a Manifest or the pixels of an image. The value _MUST_ be drawn from the set of [Creative Commons][cc-licenses] licenses, the [RightsStatements.org][rs-terms] rights statements, or those added via the [extension][ldce] mechanism. The values drawn from Creative Commons and RightsStatements.org are URIs. The inclusion of this property is informative, and for example could be used to display an icon representing the rights assertions.

If displaying rights information directly to the user is a requirement, or a publisher defined label is needed, then it is _RECOMMENDED_ to include the information using the `requiredStatement` property or in the `metadata` property.

The value _MUST_ be a string.

Any resource type _MAY_ have the `rights` property.
Clients _MAY_ process `rights` on any resource type.

**Comment by azaroth42** [comment]
Closed by #1504
